Neos.ContentGraph.DoctrineDbalAdapter/src/Domain/Repository/ContentSubgraph.php

```diff
@@ -101,18 +87,21 @@
  */
 final class ContentSubgraph implements ContentSubgraphInterface
 {
-    private int $dynamicParameterCount = 0;
+    private readonly NodeQueryBuilder $nodeQueryBuilder;
 
     public function __construct(
         private readonly ContentRepositoryId $contentRepositoryId,
+        /** @phpstan-ignore-next-line  */
+        private readonly WorkspaceName $workspaceName,
```

Comment:
Not used anywhere right now but I figured we _need_ it for creating Nodes in the next changes that follow on this, so I thought it makes sense to add it already 🤷